oauthlib/oauth2/rfc6749/grant_types/implicit.py

```diff
@@ -366,13 +363,23 @@ def validate_token_request(self, request):
                 'request': request,
         })
 
+        self._run_custom_validators(request, request_info,
+                            self._auth_validators_run_after_standard_ones,
+                            self._token_validators_run_after_standard_ones)
+
+        return request.scopes, request_info
+
+
+    def _run_custom_validators(self,
+                               request,
+                               request_info,
+                               auth_validators,
+                               token_validators):
         # For implicit grant, auth_validators and token_validators are
         # basically equivalent since the token is returned from the
         # authorization endpoint.
-        for validator in chain(self._auth_validators_run_after_standard_ones,
-                               self._token_validators_run_after_standard_ones):
+        for validator in chain(auth_validators, token_validators):
             result = validator(request)
             if result is not None:
                 request_info.update(result)
-
-        return request.scopes, request_info
+        return request_info
```

Comment:
This method returns `request_info`, but this return value is not used in any of the places where it is called.